admin_tests/preprint_providers/test_views.py

```diff
@@ -88,6 +90,35 @@ def test_update_share_token_and_source(self):
         assert self.preprint_provider.access_token == token
         assert self.preprint_provider.share_source == source_name
 
+    @responses.activate
+    @mock.patch('api.share.utils.settings.SHARE_ENABLED', True)
+    @mock.patch.object(settings, 'SPAM_FLAGGED_MAKE_NODE_PRIVATE', 'testenv')
```

Comment:
Drive-by: Should this be `SHARE_PROVIDER_PREPEND`?

Reply:
Also, if so, how did tests pass?

Reply:
I believe this actually *should* pass, but only because the response mocking includes `testenv_`, mocking `SPAM_FLAGGED_MAKE_NODE_PRIVATE` as `testenv` is just a repetitive naming choice. I think if you refactor to `@mock.patch.object(settings, 'SPAM_FLAGGED_MAKE_NODE_PRIVATE', True)` it will still work. 

The thing this test is missing though is that you don't assert that `longTitle` is sent properly in the request, only that it's received in the response properly causing some of this assertion confusion.